multisig/src/main.rs

```diff
@@ -51,12 +51,13 @@ impl Multisig {
         }
     }
 
-    pub fn add_transaction(&mut self, ctx: &Context, destination: Address, value: u64, data: Vec<u8>) -> Result<u32, Error> {
+    pub fn add_transaction(&mut self, ctx: &Context, destination: String, value: u64, data: Vec<u8>) -> Result<u32, Error> {
```

Comment:
it should be `Address`. Also you can make `data: &[u8]`

Reply:
Is there an advantage to having `data: &[u8]`?

Reply:
Also I'd rather keep it as a String for the destination since I think it would help make it more intuitive to users. I'm imagining someone deploying their service, seeing the address in the output, then copying and pasting the address in string format to the parameter.

Reply:
`data: &[u8]` is more efficient.

> keep it as a String for the destination 

except that it's easy to convert string to the more efficient byte array. it costs gas to do the parsing and sending

Reply:
I started getting the following error when setting `data: &[u8]`
```
code: Message("invalid type: sequence, expected a borrowed byte array")
```